ksql-engine/src/test/java/io/confluent/ksql/IntegrationTestHarness.java

```diff
@@ -199,13 +211,13 @@ public static void main(String args[]) throws Exception {
 
     KsqlContext ksqlContext = new KsqlContext(testHarness.ksqlConfig.getKsqlStreamConfigProps());
     ksqlContext.sql("CREATE STREAM orders (ORDERTIME bigint, ORDERID varchar, ITEMID varchar, "
-                    + "ORDERUNITS double, PRICEARRAY array<double>, KEYVALUEMAP map<varchar, double>) WITH (kafka_topic='TestTopic', value_format='JSON');");
+            + "ORDERUNITS double, PRICEARRAY array<double>, KEYVALUEMAP map<varchar, double>) WITH (kafka_topic='TestTopic', value_format='JSON');");
     ksqlContext.sql("CREATE STREAM bigorders AS SELECT * FROM orders WHERE ORDERUNITS > 40;");
 
     Map<String, GenericRow> results = testHarness.consumeData("BIGORDERS",
-                                                                         orderDataProvider.schema
-                                                                             (), 4,
-                                                                         new StringDeserializer());
+            orderDataProvider.schema
```

Comment:
for the line above, the `results` variable is never used some might aswell remove it?